templates/components/amp/common/share.html

```diff
@@ -3,11 +3,7 @@
         {{#each settings.add_this.buttons}}
             {{#all (if service '!==' 'google') (if service '!==' 'print') (if service '!==' 'facebook')}}
                 <li class="socialLinks-item socialLinks-item--{{service}}">
-                  {{#if service '===' 'google_plusone'}}
-                        <amp-social-share class="amp-social-share" data-vars-product-link="{{url}}" data-vars-product-id="{{id}}" data-vars-product-name="{{name}}" data-vars-social-type="gplus" type="gplus"></amp-social-share>
-                    {{else}}
-                        <amp-social-share class="amp-social-share" data-vars-product-link="{{url}}" data-vars-product-id="{{id}}" data-vars-product-name="{{name}}" data-vars-social-type="{{service}}" type="{{service}}"></amp-social-share>
-                  {{/if}}
+                    <amp-social-share class="amp-social-share" data-vars-product-link="{{url}}" data-vars-product-id="{{id}}" data-vars-product-name="{{name}}" data-vars-social-type="{{service}}" type="{{service}}"></amp-social-share>
```

Comment:
It looks like you are blending this change with https://github.com/bigcommerce/cornerstone/pull/1526. Can you separate them?